src/components/project/project-member/project-member.service.ts

```diff
@@ -44,49 +33,15 @@ import { ProjectMemberRepository } from './project-member.repository';
 @Injectable()
 export class ProjectMemberService {
   constructor(
-    private readonly db: DatabaseService,
-    private readonly config: ConfigService,
     @Inject(forwardRef(() => UserService))
     private readonly userService: UserService & {},
-    private readonly eventBus: IEventBus,
     @Inject(forwardRef(() => ProjectService))
     private readonly projectService: ProjectService & {},
     @Logger('project:member:service') private readonly logger: ILogger,
     private readonly privileges: Privileges,
     private readonly repo: ProjectMemberRepository,
   ) {}
 
-  protected async verifyRelationshipEligibility(
-    projectId: ID,
-    userId: ID,
-  ): Promise<void> {
-    const result = await this.repo.verifyRelationshipEligibility(
```

Comment:
if you move the exception handling to the repo, then this can be modified to: 
_await this.repo.verifyRelationshipEligibility..._

Reply:
@adam-soltech , just remembered you mentioned that this method in the service file is not being called, so prob can ignore this piece, but may want to migrate the exception handling to the repo, see comment below.

Reply:
Well, it turns out that in '22 I made the mistake of removing the call to the `verifyRelationshipEligibility` service method.  🤦🏼 

It _should_ be used right now.  So that means that yes, you should move those three checks down into the `verifyRelationshipEligibility` repo method as @atGit2021 suggested!